merge_array/merge_array.js

```diff
@@ -0,0 +1,10 @@
+function mergeArr (array1, array2) {
+  let connecArr = array1.concat(array2);
+  let newArr = [];
+  for (let i = 0; i < connecArr.length; i++){
```

Comment:
try to use `filter` function here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
@full-profane you forgot about this one